deep_qa/training/text_trainer.py

```diff
@@ -137,9 +137,11 @@ def __init__(self, params: Dict[str, Any]):
 
         self.encoder_params = params.pop('encoder', {'default': {}})
         self.encoder_fallback_behavior = params.pop('encoder_fallback_behavior', 'crash')
-        self.seq2seq_encoder_params = params.pop('seq2seq_encoder', {'default': {"encoder_params": {},
-                                                                                 "wrapper_params": {}}})
-        self.seq2seq_encoder_fallback_behavior = params.pop('seq2seq_encoder_fallback_behavior', 'crash')
+        self.seq2seq_encoder_params = params.pop('seq2seq_encoder',
+                                                 {'default': {"encoder_params": {},
+                                                              "wrapper_params": {}}})
+        self.seq2seq_encoder_fallback_behavior = \
+            params.pop_choice_with_default('seq2seq_encoder_fallback_behavior', 'crash')
```

Comment:
Looks like this is wrong - you're not passing a list of choices here, so this will actually return `'c'` as the default.